<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserProfileWidget</class>
 <widget class="QWidget" name="UserProfileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="fm_personal_infor">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>500</width>
     <height>160</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lb_major_txt">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_number">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_username">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_credit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_sex_txt">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_number_txt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_major">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>55</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>major</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_sex">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_credit_txt">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_currentborrow">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>230</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>当前借阅</string>
   </property>
  </widget>
  <widget class="QFrame" name="fm_book">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>970</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lb_bookname">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>书名</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_author">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>作者</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_press">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>出版社</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_ISBN">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ISBN</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_borrowtime">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>借阅时间</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
